core/controlplane/config/templates/cloud-config-worker

```diff
@@ -416,14 +428,15 @@ ssh_authorized_keys:
       content: |
         [Unit]
         Description=Pull and tag a mirror image for pause-amd64
-        After=docker.service
```

Comment:
This was my first choice but without "retry bash -c" . For unknown reasons, in latest CoreOS - stable version, the service start it's never called. The only way to start the service is to add it as dependency to another service. The "After=docker.service" line probably should be kept, as using only "Wants=docker.service" could fail. I will try again with this setup and make the necessary changes. In the previous version of CoreOs it was working fine.